### PR TITLE
doc improvement: Update Zigbee memory requirements page

### PR DESCRIPTION
**Pull request by sebastiandraus**
This commit updates memory usage values and adds memory requirements table for nRF21540 DK.

### CODE REVIEW

**Comment by sebastiandraus** [comment]
PR not yet complete, need one additional update

**Comment by sebastiandraus** [comment]
Updates to final version

**Review by greg-fer — APPROVED** [review]

**Comment by greg-fer** [comment]
Removing the changelog label. I'll update the release notes in #5534

File: doc/nrf/ug_zigbee_memory.rst [hunk 1]
```diff
@@ -115,12 +148,30 @@ These measurements were carried out with :ref:`zephyr:thread_analyzer`, using th
...
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+      | :ref:`Light bulb <zigbee_light_bulb_sample>`                   | 696                           | 1024                         | 1248                           | 1952                          |
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+      | :ref:`Light switch <zigbee_light_switch_sample>`               | 432                           | 1024                         | 1232                           | 1952                          |
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+
+   .. tab:: nRF53 family
+
```
Comment (greg-fer) [inline doc/nrf/ug_zigbee_memory.rst:166]:
```suggestion

The following table lists memory requirements for the :ref:`nRF53 Series <ug_nrf5340>` devices.
```

File: doc/nrf/ug_zigbee_memory.rst [hunk 2]
```diff
@@ -115,12 +148,30 @@ These measurements were carried out with :ref:`zephyr:thread_analyzer`, using th
...
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+      | :ref:`Light bulb <zigbee_light_bulb_sample>`                   | 696                           | 1024                         | 1248                           | 1952                          |
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+      | :ref:`Light switch <zigbee_light_switch_sample>`               | 432                           | 1024                         | 1232                           | 1952                          |
+      +----------------------------------------------------------------+-------------------------------+------------------------------+--------------------------------+-------------------------------+
+
+   .. tab:: nRF53 family
```
Comment (greg-fer) [inline doc/nrf/ug_zigbee_memory.rst:165]:
```suggestion
   .. tab:: nRF53 Series
```

File: doc/nrf/ug_zigbee_memory.rst [hunk 3]
```diff
@@ -115,12 +148,30 @@ These measurements were carried out with :ref:`zephyr:thread_analyzer`, using th
...
-| :ref:`Network coordinator <zigbee_network_coordinator_sample>` | 440                         | 1024                       | 1168                         | 1952                        |
-+----------------------------------------------------------------+-----------------------------+----------------------------+------------------------------+-----------------------------+
-| :ref:`Light bulb <zigbee_light_bulb_sample>`                   | 496                         | 1024                       | 1232                         | 1952                        |
-+----------------------------------------------------------------+-----------------------------+----------------------------+------------------------------+-----------------------------+
-| :ref:`Light switch <zigbee_light_switch_sample>`               | 432                         | 1024                       | 1272                         | 1952                        |
-+----------------------------------------------------------------+-----------------------------+----------------------------+------------------------------+-----------------------------+
+.. tabs::
+
+   .. tab:: nRF52 family
```
Comment (greg-fer) [inline doc/nrf/ug_zigbee_memory.rst:153]:
```suggestion
   .. tab:: nRF52 Series
```

File: doc/nrf/ug_zigbee_memory.rst [hunk 4]
```diff
@@ -115,12 +148,30 @@ These measurements were carried out with :ref:`zephyr:thread_analyzer`, using th
...
+
```
Comment (greg-fer) [inline doc/nrf/ug_zigbee_memory.rst:154]:
```suggestion

The following table lists memory requirements for the :ref:`nRF52 Series <ug_nrf52>` devices.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by totyz — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**